cmd/openshift-install/imagebased.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newImageBasedCmd(ctx context.Context) *cobra.Command {
+	imagebasedCmd := &cobra.Command{
+		Use:   "image-based",
+		Short: "Commands for supporting cluster installation using the Image-based installer",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return cmd.Help()
+		},
+	}
+
+	imagebasedCmd.AddCommand(newImageBasedCreateCmd(ctx))
+	return imagebasedCmd
+}
+
+func newImageBasedCreateCmd(ctx context.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Commands for generating image-based installer artifacts",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return cmd.Help()
+		},
+	}
+
+	cmd.AddCommand(createImageConfigTemplateCmd())
+	cmd.AddCommand(createImageCmd())
+	cmd.AddCommand(createConfigTemplateCmd())
+	cmd.AddCommand(createConfigImageCmd())
+
+	return cmd
+}
+
+func createImageConfigTemplateCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "image-config-template",
+		Short: "Generates a template of the Image-based Installation ISO config manifest used by the Image-based installer",
+		Args:  cobra.ExactArgs(0),
+		Run: func(_ *cobra.Command, _ []string) {
+			logrus.Info("Create image config template command")
+		},
+	}
+}
+
+func createImageCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "image",
+		Short: "Generates a bootable ISO image containing all the information needed to deploy a cluster",
+		Args:  cobra.ExactArgs(0),
+		Run: func(_ *cobra.Command, _ []string) {
+			logrus.Info("Create image command")
+		},
+	}
+}
+
+func createConfigTemplateCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "config-template",
+		Short: "Generates a template of the Image-based Config ISO config manifest used by the Image-based installer",
```

Comment:
Is this in addition to the install-config?

Reply:
Yes, this is the additional configuration for the IBI configuration ISO.